federation-js/src/composition/compose.ts

```diff
@@ -176,6 +183,34 @@ export function buildMapsFromServiceList(serviceList: ServiceDefinition[]) {
             );
           }
         }
+
+        for (const field of definition.fields ?? []) {
+          const fieldName = field.name.value;
+          const tagDirectives = findDirectivesOnNode(field, 'tag');
+
+          // TODO: we can dry this up into one step for both cases
```

Comment:
Yeah, if we drop the concern for `@inaccessible` being repeatable it seems the code would be exactly the same.

Reply:
26c787c2